### PR TITLE
CEDS-2036 Move reads to correct class

### PR DESCRIPTION
**Pull request by bambuchaAdm**

### CODE REVIEW

File: app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala [hunk 1]
```diff
@@ -18,10 +18,12 @@ package uk.gov.hmrc.exports.controllers.request
 
 import java.time.Instant
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{__, Json, OFormat, Reads}
```
Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala]: Is it double underscore?

File: app/uk/gov/hmrc/exports/models/declaration/ExportsDeclaration.scala [hunk 2]
```diff
@@ -50,154 +51,14 @@ case class ExportsDeclaration(
...
-      transportInformation,
-      parties,
-      locations,
-      items,
-      totalNumberOfItems,
-      previousDocuments,
-      natureOfTransaction
-    )
-  }
+  private val logger = LoggerFactory.getLogger("declaration.serializers")
```
Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/models/declaration/ExportsDeclaration.scala]: Not sure that this logger is necessary

File: app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala (hunk 1 above)
Comment (bambuchaAdm) [reply app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala]: yes

**Comment by PatrykRudnicki** [comment]
@bambuchaAdm Compilation failed

**Pull request merged**